microsoft/botframework-solutions
# first proactive scenario

## Description
This change is to enable the first proactive scenario, which is when car sends a CarStart event the bot starts checking if there's a meeting coming up in the next 60 minutes.

## Related Issue

#66 
#103 

## Testing Steps
after building and running the bot, starts with 'what's my meeting today' after finishing the dialog, send 
/event:{ "Name": "CarStart" } 
event to the bot, the bot will start checking for any meeting in the next 60 minutes for 30 minutes. then go to your outlook email account to create a meeting with someone within the next 60 minutes, then you'll see the meeting notification appearing in the chat window

## Checklist
- [ x ] I have commented my code, particularly in hard-to-understand areas
- [ x ] I have added or updated the appropriate unit tests
- [ x ] I have tested any new/updated dialogs using Speech in the emulator to ensure the [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) property is set to enable a high quality speech-first experience and the appropriate [InputHints](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) are set correctly. 

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
